docs/sources/writing-guide/tooling-and-workflows/run-a-local-webserver/index.md

```diff
@@ -1,21 +1,71 @@
 ---
-title: Run a local documentation webserver with the make-docs script
-menuTitle: Run a local documentation webserver
-description: Run a local documentation web server with the make-docs script.
+title: Run a local documentation webserver
+description: Run a local documentation webserver
 ---
 
-# Run a local documentation webserver with the make-docs script
+# Run a local documentation webserver
 
-The [`make-docs`](https://github.com/grafana/writers-toolkit/blob/main/scripts/make-docs) script can be used to mount multiple documentation sets in a single build.
-It is used by the `make docs` Make target in project repositories and can also be run separately.
+Every project keeps technical documentation in the `docs/sources` directory.
+Additionally, every project uses [GNU Make](https://www.gnu.org/software/make/) to perform tasks related to technical documentation.
+To learn more about GNU Make, refer to [GNU Make Manual](https://www.gnu.org/software/make/manual/).
 
-## Examples
+To see a list of targets and their descriptions, run `make` from the `docs/` directory.
+The output is similar to the following:
+
+```console
+Usage:
+  make <target>
+
+Targets:
+  help                                           Display this help.
+  docs-rm                                        Remove the docs container.
+  docs-pull                                      Pull documentation base image.
+  make-docs                                      Fetch the latest make-docs script.
+  docs                                           Serve documentation locally.
+  docs-no-pull                                   Serve documentation locally without pulling the latest docs-base image.
```

Comment:
```suggestion
  docs-no-pull                                   Serve documentation locally without pulling the latest `docs-base` Docker image.
```

Reply:
I'd prefer "container image" if we are gonna be specific.